### PR TITLE
fix: call getDolGlobal in modulebuilder

### PR DESCRIPTION
**Pull request by FHenry**
fix: call getDolGlobal in modulebuilder

### CODE REVIEW

File: htdocs/modulebuilder/template/class/myobject.class.php [hunk 1]
```diff
@@ -832,7 +833,7 @@ public function getNomUrl($withpicto = 0, $option = '', $notooltip = 0, $morecss
 					$pospoint = strpos($filearray[0]['name'], '.');
 
 					$pathtophoto = $class.'/'.$this->ref.'/thumbs/'.substr($filename, 0, $pospoint).'_mini'.substr($filename, $pospoint);
-					if (empty($conf->global->{strtoupper($module.'_'.$class).'_FORMATLISTPHOTOSASUSERS'})) {
+					if (empty(getDolGlobalString(strtoupper($module.'_'.$class).'_FORMATLISTPHOTOSASUSERS'))) {
```
Comment (eldy) [inline htdocs/modulebuilder/template/class/myobject.class.php:836]: empty must be replaced with !

Comment (FHenry) [reply htdocs/modulebuilder/template/class/myobject.class.php]: Fix

File: htdocs/modulebuilder/template/core/modules/mymodule/doc/doc_generic_myobject_odt.modules.php [hunk 2]
```diff
@@ -321,7 +321,7 @@ public function write_file($object, $outputlangs, $srctemplatepath, $hidedetails
 				$contactobject = null;
 				if (!empty($usecontact)) {
 					// We can use the company of contact instead of thirdparty company
-					if ($object->contact->socid != $object->thirdparty->id && (!isset($conf->global->MAIN_USE_COMPANY_NAME_OF_CONTACT) || !empty($conf->global->MAIN_USE_COMPANY_NAME_OF_CONTACT))) {
+					if ($object->contact->socid != $object->thirdparty->id && ) {
```
Comment (eldy) [inline htdocs/modulebuilder/template/core/modules/mymodule/doc/doc_generic_myobject_odt.modules.php:324]: Still a syntax error reported by travis here...

Comment (FHenry) [reply htdocs/modulebuilder/template/core/modules/mymodule/doc/doc_generic_myobject_odt.modules.php]: fix

File: htdocs/modulebuilder/template/core/modules/mymodule/doc/pdf_standard_myobject.modules.php [hunk 3]
```diff
@@ -172,32 +172,32 @@ public function write_file($object, $outputlangs, $srctemplatepath = '', $hidede
...
-		if (!empty($conf->global->MAIN_USE_FPDF)) {
+		if (getDolGlobalInt('MAIN_USE_FPDF')) {
 			$outputlangs->charset_output = 'ISO-8859-1';
 		}
 
 		// Load translation files required by the page
 		$outputlangs->loadLangs(array("main", "bills", "products", "dict", "companies"));
 
-		if (!empty($conf->global->PDF_USE_ALSO_LANGUAGE_CODE) && $outputlangs->defaultlang != $conf->global->PDF_USE_ALSO_LANGUAGE_CODE) {
+		if (!empty(getDolGlobalString('PDF_USE_ALSO_LANGUAGE_CODE')) && $outputlangs->defaultlang != getDolGlobalString('PDF_USE_ALSO_LANGUAGE_CODE')) {
```
Comment (eldy) [inline htdocs/modulebuilder/template/core/modules/mymodule/doc/pdf_standard_myobject.modules.php:182]: !empty on a result of a function seems wrong.
Do you mean if (getDol... ?

**Comment by FHenry** [comment]
@eldy it's ok now I've check all "empty(getDol" And it's ok now

**Pull request merged**